.ci_support/linux_64_c_compiler_version11cuda_compilernvcccuda_compiler_version11.8cxx_compiler_version11fortran_compiler_version11mpi_typeconda.yaml

```diff
@@ -1,29 +1,27 @@
 c_compiler:
 - gcc
 c_compiler_version:
-- '12'
+- '11'
 cdt_name:
-- cos6
+- cos7
 channel_sources:
 - conda-forge
 channel_targets:
 - conda-forge main
-cuda_version:
-- '11.2'
+cuda_compiler:
+- nvcc
+cuda_compiler_version:
+- '11.8'
```

Comment:
Instead of `cuda_version`, we now have `cuda_compiler_version`. Also moved to CUDA 11.8 as requested

Though could do 11.2 or another version just as easily. Just need to update the `skip` in the recipe